modules/apps/transfer/ibc_module.go

```diff
@@ -112,19 +112,23 @@ func (im IBCModule) OnChanOpenTry(
 	counterparty channeltypes.Counterparty,
 	counterpartyVersion string,
 ) (string, error) {
+	logger := im.keeper.Logger(ctx)
 	if err := ValidateTransferChannelParams(ctx, im.keeper, order, portID, channelID); err != nil {
 		return "", err
 	}
 
-	if counterpartyVersion != types.Version {
-		return "", errorsmod.Wrapf(types.ErrInvalidVersion, "invalid counterparty version: expected %s, got %s", types.Version, counterpartyVersion)
-	}
-
 	// OpenTry must claim the channelCapability that IBC passes into the callback
 	if err := im.keeper.ClaimCapability(ctx, chanCap, host.ChannelCapabilityPath(portID, channelID)); err != nil {
 		return "", err
 	}
 
+	if counterpartyVersion != types.Version {
+		// Propose the current version
+		logger.Debug(fmt.Sprintf("invalid counterparty version: expected %s, got %s", types.Version, counterpartyVersion))
+		logger.Debug(fmt.Sprintf("proposing the current version: %s", types.Version))
```

Comment:
we could consolidate these to one log statement, what do you think? I think you can also pass k/v args to logger after the string.

We could do something like this, but also open to using string formatting.

```suggestion
		im.keeper.Logger(ctx).Debug("invalid counterparty version, proposing current app version", "counterpartyVersion", counterpartyVersion, "version", types.Version)
```

Reply:
done